11_FFTBloom/main.cpp

```diff
@@ -955,12 +956,11 @@ class FFTBloomApp final : public examples::SimpleWindowedApplication, public app
 		pushConstants.imageTwoPixelSize_x = 4.f * imageHalfPixelSize.x;
 
 		// Interpolate between dirac delta and kernel based on current time
-		auto nextEpochSeconds = clock_t::now().time_since_epoch().count() / 1000000000.f;
-		pushConstants.interpolatingFactor = cos(nextEpochSeconds) * cos(nextEpochSeconds);
-
+		auto nextEpochNanoSeconds = clock_t::now().time_since_epoch().count();
 		// Show ms per frame
-		std::cout << "mspf: " << (nextEpochSeconds - epochSeconds) * 1000 << std::endl;
-		epochSeconds = nextEpochSeconds;
+		std::cout << "mspf: " << (nextEpochNanoSeconds - epochNanoseconds) / 1000000.f << std::endl;
```

Comment:
useless metric actually, if you want to show perf, use a timestamp query

Reply:
what's that

Reply:
you may want to check with `master` for Query Pool fixes (typos)

Reply:
> what's that

ask @Przemog1 to show you.